tox_docker.py

```diff
@@ -6,6 +6,16 @@
 from docker.errors import ImageNotFound
 import docker as docker_module
 
+            
+def escape_env_var(varname):
```

Comment:
Can you either add a test with realistic examples (preferred) or a docstring explaining why this is necessary?

Reply:
I will both.